### PR TITLE
Ltfoa daecher fixes

### PR DESCRIPTION
**Pull request by AFoletti**

### CODE REVIEW

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -1407,7 +1407,7 @@ msgid "ch.are.reisezeit-miv"
 msgstr "Time of travel IMT 2005"
 
 msgid "ch.are.reisezeit-oev"
-msgstr "Time of travel with PT 2005"
+msgstr ""
```
Comment (loicgasser) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: ?

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -3378,7 +3378,7 @@ msgid "ch.bakom.versorgungsgebiet-tv.prog"
 msgstr "Channels"
 
 msgid "ch.bakom.versorgungsgebiet-ukw"
-msgstr "Radio coverage areas"
+msgstr ""
```
Comment (loicgasser) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: ?

Comment (loicgasser) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: If not used anymore please remove the transaltion

**Comment by AFoletti** [comment]
the translation problem of the previous version was because of a couple of removed text strings in geocat

**Comment by loicgasser** [comment]
GeoCat :)

**Pull request merged**